src/rust/engine/nailgun/src/client.rs

```diff
@@ -140,13 +140,14 @@ pub async fn client_execute(
       "Client exited before the server's result could be returned." => {
         "The pantsd process was killed during the run.\n\nIf this was not intentionally done by you, \
           Pants may have been killed by the operating system due to memory overconsumption \
-          (i.e. OOM-killed). You can set the global option `--pantsd-max-memory-usage` to reduce \
-          Pantsd's memory consumption by retaining less in its in-memory cache \
-          (run `./pants help-advanced global`). You can also disable pantsd with the global \
-          option `--no-pantsd` to avoid persisting memory across Pants runs, although you will miss \
-          out on additional caching.\n\nIf neither of those help, please consider filing a \
-          GitHub issue or reaching out on Slack so that we can investigate the possible memory \
-          overconsumption (https://www.pantsbuild.org/docs/getting-help)."
+          (i.e. OOM-killed). If you keep seeing this error message, try the troubleshooting steps below. \
+          If none of those help, please consider filing a GitHub issue or reaching out on Slack so that we can \
+          investigate the possible memory overconsumption (https://www.pantsbuild.org/docs/getting-help).\n \
+          - Exit all other applications, including applications running in the background.\n \
```

Comment:
It may be sufficient in many cases to only close a few. That is a better experience for users.

```suggestion
          - Exit other applications, including applications running in the background.\n \
```

Reply:
You're right, it may be sufficient to only close a few. Users can use their discretion to close applications as needed.